deploy.bash

```diff
@@ -83,7 +83,7 @@ fi
 
 echo "2. Install cpan packages... "
 if [[ $step =~ (^|,)2(,|$) || $step == "*" ]]; then
-	cpan_packages=(DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
+	cpan_packages=(inc::Module::Install DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
```

Comment:
`cpanm` should already install all required dependencies. What is special about this one?

Reply:
```
cpanm Text::MultiMarkdown
...
Configuring Text-MultiMarkdown-1.000035 ... N/A
! Configure failed for Text-MultiMarkdown-1.000035. See /home/keltar/.cpanm/work/1527485029.3006/build.log for details.
```
build.log:
```
...
Searching Text::MultiMarkdown () on cpanmetadb ...
--> Working on Text::MultiMarkdown
Fetching http://www.cpan.org/authors/id/B/BO/BOBTFISH/Text-MultiMarkdown-1.000035.tar.gz
-> OK
Unpacking Text-MultiMarkdown-1.000035.tar.gz
Entering Text-MultiMarkdown-1.000035
Checking configure dependencies from META.yml
Checking if you have ExtUtils::MakeMaker 6.59 ... Yes (7.24)
Configuring Text-MultiMarkdown-1.000035
Running Makefile.PL
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /home/keltar/perl5/lib/perl5/5.26.0/x86_64-linux-gnu-thread-multi /home/keltar/perl5/lib/perl5/5.26.0 /home/keltar/perl5/lib/perl5/x86_64-linux-gnu-thread-multi /home/keltar/perl5/lib/perl5 /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.0 /usr/local/share/perl/5.26.0 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at Makefile.PL line 2.
BEGIN failed--compilation aborted at Makefile.PL line 2.
...
```

Reply:
Change inc::Module::Install to Module::Install.